frontend/coprs_frontend/coprs/views/webhooks_ns/webhooks_general.py

```diff
@@ -92,14 +92,16 @@ def add_webhook_history_record(webhook_uuid, user_agent='Not Set',
         log.debug("No build initiated. Webhook not logged to db.")
         return
 
-    webhookRecord = models.WebhookHistory(created_on=int(time.time()),
+    webhook_record = models.WebhookHistory(created_on=int(time.time()),
                                           webhook_uuid=webhook_uuid,
                                           user_agent=user_agent)
-    db.session.add(webhookRecord)
-    db.session.commit()
+    db.session.add(webhook_record)
+
+    if not isinstance(builds_initiated_via_hook, list):
+        builds_initiated_via_hook = [builds_initiated_via_hook]
```

Comment:
Ah, I missed this before.  Can we simply expect just a list input?  This check is not complete and ideal (what if None comes in), and it makes the "method API" complicated.

Reply:
fixed ty :)